client/src/pages/Landing/Landing.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React, { useContext } from 'react';
```

Comment:
No need for useContext here it can be removed.  

also the console.log() after defining user can be removed.